lib/features/family/features/reflect/presentation/pages/guess_secret_word_screen.dart

```diff
@@ -95,40 +95,44 @@ class _GuessSecretWordScreenState extends State<GuessSecretWordScreen> {
                     itemCount: uiModel.guessOptions.length,
                     itemBuilder: (context, index) {
                       final guessOption = uiModel.guessOptions[index];
-                      return FunTile(
-                        shrink: true,
-                        onTap: () {
-                          _cubit.onClickOption(index);
-                        },
-                        mainAxisAlignment: MainAxisAlignment.center,
-                        isPressedDown:
-                            guessOption.state == GuessOptionState.wrong,
-                        titleBig: guessOption.text,
-                        textColor: guessOption.state == GuessOptionState.initial
-                            ? FamilyAppTheme.tertiary40
-                            : guessOption.state == GuessOptionState.correct
-                                ? Colors.green
-                                : Colors.red,
-                        analyticsEvent: AnalyticsEvent(
-                          AmplitudeEvents.reflectAndShareGuessOptionClicked,
-                          parameters: {
-                            'option': guessOption.text,
+                      return Semantics(
+                        identifier: 'funtile_guess_option_$index',
```

Comment:
Is that enough or does Donna need a option to easily find the correct option? :)

Reply:
Good one, will look into it